src/release/issue-triaging.md

```diff
@@ -54,6 +54,16 @@ To see a list of all labels, check out the "labels" page next to the search bar
 [IRLO]: https://internals.rust-lang.org/
 [Discord]: https://discord.gg/rust-lang
 
+### ICE Triage
+For [Issues that have both I-ICE and needs-triage](https://github.com/rust-lang/rust/issues?q=is%3Aissue%20state%3Aopen%20label%3AI-ICE%20label%3Aneeds-triage)
+* If it an older (like latest stable) version of rust, ask for or check the latest nightly (potentially check for duplicates)
+* If it does not have a reproduction, comment asking for one and add S-needs-repro. if there isn't one for around a month it should generally be closed.
```

Comment:
Suggestion: check also if it's a regression:

- Especially double check if it's a regression-from-stable-to-{stable,beta,nightly}, or if unclear add `regression-untriaged`.
- Check if this needs prioritization.
- If it is a regression and there is a reproducer that's friendly to bisection within reasonable bisection step time, then tag `E-needs-bisection` if not already bisected and indicate to other people to aid with bisection, or run one yourself.
- If the bisection is able to pinpoint to a particular commit, double check if the commit roughly seems to be possibly correlated with the ICE itself -- sometimes the change can be more like "report more errors" which is not really the "root cause" of the ICE